src/main/groovy/org/openmastery/publisher/core/IdeaFlowInMemoryPersistenceService.groovy

```diff
@@ -229,14 +231,14 @@ public class IdeaFlowInMemoryPersistenceService implements IdeaFlowPersistenceSe
 	}
 
 	@Override
-	List<TaskEntity> findRecentTasks(Long userId, int limit) {
-		if (limit < 1) {
+	Page<TaskEntity> findRecentTasks(Long userId, int page, int pageSize) {
```

Comment:
Just noticed that I named this one 'pageSize' instead of 'perPage' like the others. I will correct this.